### PR TITLE
Stay up to date with Rails `main` branch

### PR DESCRIPTION
**Pull request by etagwerker**
Hi there, 

This is a PR to make sure that we run the test suite using the latest changes in Rails `main`.

It is part of our work at: https://www.fastruby.io/stay-up-to-date

Please check it out. 

Thanks!

### CODE REVIEW

**Comment by cleicar** [comment]
@etagwerker is this still a WIP? I see the CI is failing

**Comment by etagwerker** [comment]
@cleicar This is now ready for review. The PR is now passing with both Rails 6.1 and Rails main. 👍

**Comment by etagwerker** [comment]
Not sure why, but in recent versions of Octokit it is not raising an Octokit::NotFound error, so I had to fix the way closed issues are removed: https://github.com/fastruby/dash/pull/64/commits/14e882ca5031e4235bf7824234a9fc2a2e4ca415

File: .circleci/config.yml [hunk 1]
```diff
@@ -112,7 +112,7 @@ jobs:
             gem update bundler
             bundle update --bundler
             export BUNDLE_GEMFILE=Gemfile.next
-            bundle install --jobs=4 --retry=3 --path vendor/bundle
+            bundle update --jobs=4 --retry=3
```
Comment (bronzdoc) [inline .circleci/config.yml:115]: I'm comfused about this change, we will not install the gems but update them everytime this job runs?

Comment (bronzdoc) [reply .circleci/config.yml:115]: Ah forgot that bundle update installs stuff... Do we still want this? Isn't this job updating gems everytime could lead to underterministic results for the CI?

Comment (etagwerker) [reply .circleci/config.yml:115]: @bronzdoc This is core to the "stay up to date" service: https://www.fastruby.io/stay-up-to-date

Considering that we want to get the latest from Rails `main` and then run the test suite against that, I think this is the best way to do it. I don't think `bundle install` will go and fetch the latest changes with `bundle install` 

The unique value proposition of our service is that we will figure out what changes in Rails `main` will break your application code (_undeterministic results_) and fix them. 😸

Comment (bronzdoc) [reply .circleci/config.yml:115]: Make sense, thanks for the clarification @etagwerker

**Review by bronzdoc — APPROVED** [review]

**Pull request merged**